admin-src/utils/index.js

```diff
@@ -20,3 +20,18 @@ export const getDownloadURLFromImages = async (imageReferences) => (
     Promise.resolve([])
   )
 )
+
+export const userHasAccess = async () => {
+  try {
+    const user = await firebase.auth().currentUser
+
+    if (!user) return false
+
+    const result = await user.getIdTokenResult()
+
+    return Boolean(result.claims.isAdmin)
+  } catch (error) {
+    console.error(error.message)
+    return false
```

Comment:
`console.error` and the like of them are commonly used to help us in development to get additional information to debug. When an error occurs, we need to solve it or provide some usable alternative to the consumer of the code.

In my eyes, this console.error is not necessary, as you correctly solve the situation.
This is nitpicky but it would be good if we decide on these things for consistency. For example, if we see a console.error we are sure that something was not taken care of properly as we require that debug information. We can always set an error to show to the user or just set an error state.

@nahudalla

Reply:
I agree, no need to log the error if you handled it (not allowing access).